src/Kuzzle.ts

```diff
@@ -857,7 +857,7 @@ Discarded request: ${JSON.stringify(request)}`));
    * @param options Request options
    * @returns Resolved request or a TimedOutError
    */
-  private _timeoutRequest(delay: number, request:JSONObject, options:JSONObject = {}) {
+  private _timeoutRequest(delay: number, request:RequestPayload, options:JSONObject = {}) {
```

Comment:
Nitpicking: coding standard
```suggestion
  private _timeoutRequest(delay: number, request: RequestPayload, options: JSONObject = {}) {
```